packages/foreman/rubygem-actionpack/rubygem-actionpack.spec

```diff
@@ -17,7 +17,7 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 2.2.2
```

Comment:
This is out of date if I look at https://rubygems.org/gems/actionpack/versions/6.0.2.1

Reply:
I'll go and fix this for all these packages.